gov-notify-stub/gov-notify-spec.yaml

```diff
@@ -47,6 +47,55 @@ paths:
         passthroughBehavior: "when_no_match"
         contentHandling: "CONVERT_TO_TEXT"
         type: "aws_proxy"
+
+  /govnotify/v2/notifications/letter:
+    post:
+      tags:
+        - Backend Endpoint for GovNotify
+      summary: Mock GovNotify's Send letter endpoint
+      requestBody:
```

Comment:
No need to provide all the params of requestBody here if its defined in the schema,
requestBody:
        description: SendLetterPayload JSON object (any other suitable description)
        required: true (Please evaluate if this needs to be set to true/false)
        content:
          application/json:
            schema:
              $ref: "#/components/schemas/SendLetterPayload"